src/main/java/com/example/s3practical_task/configuration/AwsConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package com.example.s3practical_task.configuration;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AwsConfiguration {
+
+    @Bean
+    public AWSCredentials credentials(@Value("${aws.accessKey}") String accessKey,
+                                      @Value("${aws.secretKey}") String secretKey) {
+        return new BasicAWSCredentials(accessKey, secretKey);
+    }
+
+    @Bean
+    public AmazonS3 amazonS3(AWSCredentials awsCredentials) {
```

Comment:
take a look how chain of credential providers work and what benefits we gain from it